GUI/SearchTaskWindow.xaml

```diff
@@ -222,12 +396,62 @@
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
+                                                <LineBreak/>
+                                                This places notches around +/- 1 Da, such that the issue is accomodated if the difference is slightly off.
+                                                <LineBreak/>
+                                                Example: A spectrum with a reported precursor mass of 1000 at 0.1 Da tolerance can match to theoreticals with masses of 998.9-999.1, 999.9-1000.1, and 1000.9-1001.1.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept2mm" Content="1 or 2 Missed Monoisotopic Peaks" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
+                                                <LineBreak/>
+                                                This places notches around +/- 1 and 2 Da, such that the issue is accomodated if the difference is slightly off.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept3mm" Content="1, 2, or 3 Missed Monoisotopic Peaks" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
+                                                <LineBreak/>
+                                                This places notches around +/- 1, 2, and 3 Da, such that the issue is accomodated if the difference is slightly off.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept187" Content="-187 and Up" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
```

Comment:
An "open-mass" search that allows precursor mass-differences of -187 Da to infinity. The goal of this search type is to detect mass-differences corresponding to PTMs, amino acid variants, sample handling artifacts, etc. Use "Modern Search" when using this search type.